graph/src/data/subgraph/schema.rs

```diff
@@ -199,6 +200,22 @@ impl SubgraphDeploymentEntity {
         }
     }
 
+    /// Return true if the deployment with the given id is fully synced,
+    /// and return false otherwise. Errors from the store are passed back up
+    pub fn is_synced(
+        store: &Store,
+        id: &SubgraphDeploymentId,
+    ) -> Result<bool, QueryExecutionError> {
+        let filter = EntityFilter::new_equal("id", id.0.clone());
+        let entity = store.find_one(SubgraphDeploymentEntity::query().filter(filter))?;
```

Comment:
Ah, one more thing. There is a store query here, so it needs to have a corresponding `AbortUnless` op

Reply:
Yup you're right!